javascript/javascript2/week3/function.js

```diff
@@ -0,0 +1,24 @@
+//FUNCTION AS VARIABLE
+const fruits = () => 'banana';//return single value
+const vegetables = () => ['aubergine', 'bell-peppers']; //returning multiple values
+const salads = () => console.log('lettuce', 'kale')
+
+const functionArray = [fruits, vegetables, salads];
+
+functionArray.forEach(item => item);
+console.log(functionArray);
+
+//REGULAR FUNCTION
+//with regular function it just logs out the functon name
+function house() { console.log('apartment', 'villa') };
+function road() { console.log('cobbled', 'cemented') };
+function furniture() { console.log('sofa', 'chairs') };
+
+const newFunctionArray = [house(), road(), furniture()];
+
+newFunctionArray.forEach(item => item);
```

Comment:
Your array newFunctionArray will end up equalling [undefined, undefined, undefined] 
```suggestion
const newFunctionArray = [house, road, furniture];

newFunctionArray.forEach(item => item());
```

Reply:
oohhh yesss...I will change it